builders/builder/builder.toml

```diff
@@ -1,4 +1,4 @@
-description = "Ubuntu 24.04 Noble Numbat base image with buildpacks for Java, Node.js and Procfile"
+description = "Ubuntu 24.04 Noble Numbat base image with buildpacks for Java, Node.js, .NET and Procfile"
```

Comment:
Want me to add Nginx and HTTPD here while I'm at it? 😅

Reply:
I think hey have already been added, you just need to rebase. Thanx for thinking about it